.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: Build package
+name: Test and build package
```

Comment:
I personally prefer shorter names (both for the file and the job), as it keeps the checks list below inside the PRs tidier. Especially on smaller screens, the second half can drop off screen if names become too long.